# HG changeset patch
# Parent  db540e5d5cbf46234a4349d46a7bbbbcd0eb9c7a
Ensure we fortify properly, features.h is pulled in via fortification.

diff --git a/config/system-headers.mozbuild b/config/system-headers.mozbuild
--- a/config/system-headers.mozbuild
+++ b/config/system-headers.mozbuild
@@ -224,17 +224,16 @@ system_headers = [
     'Events.h',
     'exception',
     'exdisp.h',
     'ExDisp.h',
     'exe386.h',
     'execinfo.h',
     'extras.h',
     'fcntl.h',
-    'features.h',
     'fenv.h',
     'ffi.h',
     'fibdef.h',
     'File.h',
     'filehdr.h',
     'files.h',
     'Files.h',
     'FindDirectory.h',
diff --git a/media/webrtc/signaling/src/sdp/sipcc/sdp_os_defs.h b/media/webrtc/signaling/src/sdp/sipcc/sdp_os_defs.h
--- a/media/webrtc/signaling/src/sdp/sipcc/sdp_os_defs.h
+++ b/media/webrtc/signaling/src/sdp/sipcc/sdp_os_defs.h
@@ -15,13 +15,10 @@
 
 /* Use operating system malloc */
 #define SDP_MALLOC(x) calloc(1, (x))
 #define SDP_FREE free
 
 typedef uint8_t    tinybool;
 typedef unsigned short ushort;
 typedef unsigned long  ulong;
-#ifndef __GNUC_STDC_INLINE__
-#define inline
-#endif
 
 #endif /* _SDP_OS_DEFS_H_ */
